### PR TITLE
haproxy 1.5.10

### PR DESCRIPTION
**Pull request by DomT4**
Version bump, style nits, basic test.

### CODE REVIEW

File: Library/Formula/haproxy.rb [hunk 1]
```diff
@@ -16,13 +14,14 @@ class Haproxy < Formula
...
 
   def install
-    args = ["TARGET=generic",
-            "USE_KQUEUE=1",
-            "USE_POLL=1",
-            "USE_PCRE=1",
-            "USE_OPENSSL=1",
-            "USE_ZLIB=1",
-            "ADDLIB=-lcrypto",
+    args = %w[
```
Comment (DomT4) [inline Library/Formula/haproxy.rb]: I thought this should always be ` args = %W[ ` but the ` brew style ` command flagged it up as wrong?

Comment (mikemcquaid) [inline Library/Formula/haproxy.rb]: Nah, it's right.

File: Library/Formula/haproxy.rb [hunk 2]
```diff
@@ -30,4 +29,8 @@ def install
     man1.install "doc/haproxy.1"
     bin.install "haproxy"
   end
+
+  test do
+    system bin/"haproxy", "-v"
```
Comment (mikemcquaid) [inline Library/Formula/haproxy.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/haproxy.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/haproxy.rb]: Ah. Is it only ` args = %W[ ` when you do something like ` --prefix=#{prefix} ` or similar?

File: Library/Formula/haproxy.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/haproxy.rb]: Probably not, sadly. I could look into setting up a proxy from port 8080 to 8081 or something briefly, but I'd probably have to write a configuration file and without a way to kill it after starting it, we may end up leaving the proxy running.

Comment (mikemcquaid) [inline Library/Formula/haproxy.rb]: :cool:

**Pull request closed without merging**